eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -1,30 +1,53 @@
 parameters:
   LiveTestSamples: []
 
-
 jobs:
-  - job: LiveTest
+  - job:
+    displayName: LiveTest
     condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
     strategy:
       matrix:
         Linux_x64_with_iot_samples:
-          Pool: azsdk-pool-mms-ubuntu-2004-general
-          OSVmImage: ubuntu-20.04
+          Pool: $(LINUXPOOL)
```

Comment:
Before someone asks about needing to use the variable...I think that variable replacement only happens on the very surface level of the template construct. 

[Commit](https://github.com/Azure/azure-sdk-for-c/commit/a787af99bb9e65938c7c5b91b7b6cd69954bbc65)
[Build](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3616204&view=results)

Notice how `$(LINUXVMIMAGE)` isn't being honored? It simply isn't replacing it when the pool is one level deeper than the standard.

Reply:
No idea why, reverting the changes though.